argo/providers/kfp.mk

```diff
@@ -1,21 +1,51 @@
+include ../../common.mk
+include get-proto.mk
 IMG := kfp-operator-kfp-provider
 
 all: build
 
+##@ Build
+
+MOCKGEN := $(PROJECT_DIR)/bin/mockgen
+mockgen: ## Download mockgen locally if necessary.
+	$(call go-install,$(PROJECT_DIR)/bin/mockgen,github.com/golang/mock/mockgen@v1.6.0)
+
+generate: protoc_gen_go mockgen ## Generate service definitions from protobuf
```

Comment:
`protoc_gen_go` has a different capitalisation than the rest of the makefile.

Reply:
addressed and changed to `protoc-gen-go `